src/components/nav_bar/Navbar.test.tsx

```diff
@@ -12,8 +12,10 @@ import Navbar from './Navbar';
  * - Clicking auth options should open authentication drawer.
  */
 
+afterEach(() => cleanup);
+
 describe('Navbar', () => {
-    test('Should render at all times', () => {
+    test('should render at all times', () => {
```

Comment:
my go-to for these first tests is usually `should not throw`. This is an example of one of those silly tests, it doesn't do much but get your feet wet.

the assertion for that is slightly different, with:
```tsx
expect(() => render(
  <TestWrapper>
    <Navbar
      isUserLoggedIn={false}
      toggleAuthDrawerWithView={(state, view) => {}}
    />
  </TestWrapper>
)).not.toThrow();
```

I'm usually pretty particular about spacing on tests, too. _*Always*_ pad `expect` with an empty line above it.